spec/requests/resources_controller_spec.rb

```diff
@@ -53,4 +53,32 @@ def perform
       end
     end
   end
+
+  describe 'GET /index' do
```

Comment:
These tests would go in the learning_units_controller_spec, describing the #show.